core/cache/cachetest/state.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package cachetest
```

Comment:
In a later branch where I'm hooking up the layers. It got added to this branch because I just grabbed the whole core/cache package as a branch to create.